WISE-Community/WISE-Client
# feat(Tag): Add custom color picker

## Changes
- Added custom color picker for tags

You can see the component parameters [here](https://zefoy.github.io/ngx-color-picker/).

## Test
- Run `npm install`
- Make sure the custom color chooser works when creating a new tag and editing an existing tag
